backend/app/models.py

```diff
@@ -248,17 +248,23 @@ class DocumentDataExampleBase(SQLModel):
 class DocumentDataExampleCreate(DocumentDataExampleBase):
     document_id: str
     data: str
+    start_page: int = 0
+    end_page: int | None = None
 
 class DocumentDataExampleUpdate(DocumentDataExampleBase):
     document_id: str | None = None
     data: str | None = None
+    start_page:  int | None = None
```

Comment:
Please do not add extra spaces where not needed.
We should use a linter at some point to automatically enforce style correctness